src/views/public/login/index.js

```diff
@@ -142,9 +146,19 @@ export default function Login() {
                                     {errorFields.email ? <label className="text-danger">{errorFields.email}</label> : null}
                                 </div>
                                 <div className="form-floating">
-                                    <CustomInputFields type="password" className="form-control" id="floatingPassword" placeholder="Password" required onChange={(e) => validateInput("password", e.target.value, 40, ERROR_MESSAGES, setErrorFields)} />
+                                    <CustomInputFields   type={showPassword ? 'text' : 'password'} className="form-control" id="floatingPassword" placeholder="Password" required onChange={(e) => validateInput("password", e.target.value, 40, ERROR_MESSAGES, setErrorFields)} />
```

Comment:
text : "pas... in consts
and instead of this.
you can set setPasswordField  to text and password instead f true false